myapp/src/App.js

```diff
@@ -0,0 +1,194 @@
+import logo from './logo.svg';
+import beverly from './Images/beverly.jpeg'
+import fall from './Images/fall.jpeg'
+import plants from './Images/plants.jpeg'
+import self from './Images/self.jpeg'
+import './App.css';
+import Navbar from "./components/Navbar/Navbar"
+import ImageGallery from "./components/ImageGallery"
+import { Component } from 'react';
+import axios from 'axios'; 
+
+class App extends Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      currPage: "Home",
+      data: null
+    };
+}
+  state = {currPage: true}
+  
+
+  fetchData = () => {
```

Comment:
if you wanted to work with components more, this whole jeopardy question api call could be a separate component.